### PR TITLE
refactor(BDDs): Removing Beforescript from tests and using a separate clientset file for getting kubeconfig

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: Udit Gaurav <udit.gaurav@mayadata.io>
issue - https://github.com/litmuschaos/litmus/issues/1531
_This PR includes:_ 
- Removal of _BeforeScript_ function from each test file under test pkg this is because :
if we run all tests using `go test ./...` or tests in parallel  `ginkgo -nodes=N` it will run all the test files in a pkg at once and will error that only one BeforeScript can be used.
- Separating `GetKubeConfig()` from types.go to a new file in pkg clienset.go
- Using `	command := flag.NewFlagSet(os.Args[0], flag.ExitOnError)` in clientset.go so that for running all tests it should not give an error for repeated flags.
- Moving install litmus and uninstall litmus test to a different pkg that is litmus

### CODE REVIEW

**Comment by uditgaurav** [comment]
Impact of this:
- Change in the path of install & uninstall litmus in [github actions](https://bit.ly/2XmuqBy).
- Change in the path of install & uninstall litmus in [gitlab remote templates](https://github.com/mayadata-io/gitlab-remote-templates/blob/839c0dcba909def5181dbbba3145e21f72fda8b2/templates/install-litmus-template.yml#L10)

**Review by ksatchit — APPROVED** [review]
Agree to the move to clientset and removal of beforeSuite in principle. Would like @ispeakc0de to take a closer look. LGTM

File: tests/pod-memory-hog_test.go [hunk 1]
```diff
@@ -94,6 +79,8 @@ var _ = Describe("BDD of pod-memory-hog experiment", func() {
 			Expect(err).To(BeNil(), "Failed to update application kind in engine")
 			err = pkg.EditFile(experimentName+"-ce.yaml", "jobCleanUpPolicy: 'delete'", "jobCleanUpPolicy: 'retain'")
 			Expect(err).To(BeNil(), "Failed to update application label in engine")
+			err = pkg.EditFile(experimentName+"-ce.yaml", "name: pod-memory-hog", "name: k8s-pod-memoey-hog")
```
Comment (ksatchit) [inline tests/pod-memory-hog_test.go]: typos

File: tests/pod-network-latency_test.go [hunk 2]
```diff
@@ -94,6 +79,8 @@ var _ = Describe("BDD of pod-network-latency experiment", func() {
 			Expect(err).To(BeNil(), "Failed to update application kind in engine")
 			err = pkg.EditFile(experimentName+"-ce.yaml", "jobCleanUpPolicy: 'delete'", "jobCleanUpPolicy: 'retain'")
 			Expect(err).To(BeNil(), "Failed to update application label in engine")
+			err = pkg.EditFile(experimentName+"-ce.yaml", "name: pod-network-latency", "name: k8s-pod-network-lateny")
```
Comment (ksatchit) [inline tests/pod-network-latency_test.go]: typo

**Review by ksatchit — APPROVED** [review]

**Pull request merged**